test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,92 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const statusCode = require('http-status-codes');
+const md5 = require('md5');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+
+chai.use(chaiSubset);
+
+const { expect } = chai;
+const baseUrl = 'https://api.github.com';
```

Comment:
duplicated lines in the test files, consider using some config common file for the test cases.